libs/blocks/author-header/author-header.css

```diff
@@ -0,0 +1,71 @@
+.author-header-wrapper {
+  visibility: unset;
+  background: var(--color-gray-100);
+  border: 1px solid var(--color-gray-100);
+  text-align: center;
+}
+
+.author-header {
+  background: var(--color-gray-100);
+  border: 1px solid var(--color-gray-100);
+  margin: 3rem auto;
```

Comment:
nit: I'd change the margin to `48px auto` since you probably don't want more margin for the header if the user zooms in.